lib/Github/Api/Authorizations.php

```diff
@@ -40,6 +40,16 @@ public function remove($id)
 
     public function check($id, $token)
```

Comment:
I suggest renaming ``id`` to ``clientId`` to be more explicit about which id it is (the github doc describes it as ``client_id`` btw): https://developer.github.com/v3/oauth_authorizations/#check-an-authorization